Src/Base/AMReX_RKIntegrator.H

```diff
@@ -0,0 +1,262 @@
+#ifndef AMREX_RK_INTEGRATOR_H
+#define AMREX_RK_INTEGRATOR_H
+#include <AMReX_REAL.H>
+#include <AMReX_Vector.H>
+#include <AMReX_ParmParse.H>
+#include <AMReX_IntegratorBase.H>
+#include <functional>
+
+namespace amrex {
+
+namespace ButcherTableauTypes {
+    enum ButcherTypes {User = 0, ForwardEuler, Trapezoid, SSPRK3, RK4, NumTypes};
+};
```

Comment:
We should consider making this a strong type (i.e., `enum struct`).

Reply:
doing this now ...